docs/developer-docs/web-apps/verifiable-credentials/how-it-works.mdx

```diff
@@ -0,0 +1,84 @@
+---
+keywords: [intermediate, concept, identity, credentials, verifiable credentials]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# How it works
+
+<MarkdownChipRow labels={["Intermediate", "Concept"]} />
+
+## Overview
+
+Verifiable credentials (VCs) on ICP rely on Internet Identity (II), which provides different and unlinkable identities for each dapp. A dapp utilizing verifiable credentials can’t ask for a user's credentials from the issuer of the verifiable credential directly. When a relying party wants to obtain a credential from an issuer, it must request it through an identity provider such as Internet Identity.
```

Comment:
Small comment, the character in "can't" seens to be a different single quote than in "user's". I guess we should use `'`, right?

Reply:
Both characters render the same on the webpage, and we use a mix of both throughout the docs. The difference in character comes from which IDE you are using (VS Code vs editing directly in Github, etc).

Reply:
Ah, ok, thanks for the explanation!